checkov/main.py

```diff
@@ -52,7 +53,7 @@ def run(banner=checkov_banner):
     args = parser.parse_args()
     bc_integration = BcPlatformIntegration()
     runner_filter = RunnerFilter(framework=args.framework, checks=args.check, skip_checks=args.skip_check)
-    runner_registry = RunnerRegistry(banner, runner_filter, tf_runner(), cfn_runner(), k8_runner())
+    runner_registry = RunnerRegistry(banner, runner_filter, tf_runner(), cfn_runner(), k8_runner(), sls_runner())
```

Comment:
you probably want to add it in the argparse options https://github.com/bridgecrewio/checkov/pull/369/files#diff-9550d7646c210ba5bf61f16fe39e52b7R37